packages/backend/src/apps/formatter/actions/date-time/transforms/convert-date-time/fields.ts

```diff
@@ -47,6 +49,16 @@ export const field = {
       description: '02 January 2006',
       value: ensureZodEnumValue(formatStringsEnum, 'dd LLLL yyyy'),
     },
+    {
+      label: 'YY/MM/DD',
+      description: '24/01/22',
+      value: ensureZodEnumValue(formatStringsEnum, 'yy/LL/dd'),
+    },
+    {
+      label: 'YYYY/MM/dd',
```

Comment:
nit: should put the label as `DD` instead of `dd` for consistency?

Reply:
ya nice catch